red/red.js

```diff
@@ -36,6 +38,7 @@ var RED = {
         userSettings.version = this.version();
         settings.init(userSettings);
         server.init(httpServer,settings);
+        memoryMetricsReporter.init(); //TODO pass in interval settings
```

Comment:
@zobalogh I'm adding a setting "metricsOn" in settings.js which if set to true will report metrics, and if false (or not there) will not report metrics. Pull request containing this hopefully available by the end of the day.